mubench.pipeline/utils/io.py

```diff
@@ -157,4 +158,8 @@ def zip_dir_contents(sources: List[str], destination: str):
                 for file in files:
                     file_path = join(root, file)
                     file_path_in_archive = relpath(file_path, source)
-                    archive.write(file_path, file_path_in_archive)
+                    if not any(file.endswith(file_path_in_archive) for file in archive.namelist()):
```

Comment:
Why is this `endswith`? Isn't a conflict only when they are equal? This way, if the archive contains `foo/Bar.java` adding `Bar.java` would be rejected, wouldn't it?